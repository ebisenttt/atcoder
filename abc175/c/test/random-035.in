-80794876430301 374345951863780 476930391718631
