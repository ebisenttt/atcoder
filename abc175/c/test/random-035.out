80794876430301
